requirements-rocm.txt

```
@@ -13,3 +13,4 @@ uvicorn[standard]
 pydantic >= 2.0  # Required for OpenAI server.
 prometheus_client >= 0.18.0
 outlines == 0.0.34
+loguru
```

```suggestion
loguru==0.7.2
```

Can we pin this to avoid breaking builds later when loguru updates?